Automattic/wp-calypso
# `PluginsBrowser`: Remove breadcrumbs

Related to https://github.com/Automattic/wp-calypso/pull/92003#issuecomment-2188879267

## Proposed Changes

* Remove breadcrumbs from the plugins browser, leaving the main `Plugins marketplace` title

|Before|After|
|---|---|
|![image](https://github.com/Automattic/wp-calypso/assets/8511199/9930922a-4151-48a1-9855-18a0a2a62d3b)|![image](https://github.com/Automattic/wp-calypso/assets/8511199/96e6c9bc-dc42-48d0-9dc8-599dd214c446)|

## Testing Instructions

* Open the live preview
* Go to `/plugins`
* Click on any category, check that the header doesn't change

## Pre-merge Checklist
